hack/quickstart-setup.sh

```diff
@@ -26,7 +26,7 @@ if [ -z $MGC_REF ]; then
   MGC_REF=${MGC_REF:="main"}
 fi
 
-if [ -z $ISTIO_INSTALL_SAIL]; then
+if [ -z $ISTIO_INSTALL_SAIL ]; then
```

Comment:
I think I fixed this already in #412 so might need a rebase

Reply:
I am sorry, I did not noticed that. Rebased and force pushed.